src/main/java/nextstep/courses/domain/Image.java

```diff
@@ -0,0 +1,34 @@
+package nextstep.courses.domain;
+
+public class Image {
+    Integer size;
+    ImageType type;
+    Integer width;
+    Integer height;
```

Comment:
```suggestion
    Integer width;
    Integer height;
```

하나의 객체로 만들수 있지 않을까요.
그리고 특별한 책임을 줄 수 있지 않을까요?

Reply:
Image class의 inner class로 수정했습니다!